docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepgvector.md

```diff
@@ -23,41 +21,38 @@ You can find authentication information for this node [here](/integrations/built
 
 --8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-mode.md"
 
-### Parameters for Get Many
+### Get Many parameters
 
-* Table name
-* Prompt: search query.
-* Limit: how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+* **Table name**: Enter the name of the table you want to query.
```

Comment:
These parameters are similar across a few vector stores -- should we apply this change to all of the others too then?

Reply:
@jeanpaul if we can, yes! I have some app nodes to update formatting on already, so if you'd like I can just leave myself a note and update those as I'm updating other things. :)

Reply:
Yes please! That'd be great.